app/Bundle.php

```diff
@@ -72,27 +107,28 @@ class Bundle extends Model
     public function alterVouchers(Collection $vouchers, array $codes = [], Bundle $bundle = null)
```

Comment:
I'm not that keen on the $self/ $this reference of Bundle. Might simplify things if addVouchers took care of this... but if you are using to replace whole bundle elsewhere we need to keep?

Reply:
I'm not either, but it happened organically while i was avoiding writing the same thing everywhere